Add axes command to show cell axis directions

kmdalton opened the pull request:
This pr resolves #9 by adding an axes command that shows the directions of the cell axes A, B, C in red, green, and blue arrows.

kmdalton commented:
Hi, @speleo3, here is a very rudimentary function which draws VMD style arrows pointing in the cell axis directions. Certainly there could be some improvements. Supporting colorblind friendly arrows would be a good start. I figured it was better to get basic functionality first. Let me know what you think!

speleo3 commented:
Thanks @kmdalton, this looks good! We may want to keep the `axes` command compatible to the original https://pymolwiki.org/index.php/Axes code, e.g. don't grab `cmd.get_object_list()[0]` if object is `None`. How about adding a new command `axes_object` (or a similar name) for the new behavior?

kmdalton commented:
@speleo3 -- I renamed the function to `cell_axes`. Does that address your compatibility concern?

I also added optional arguments for axis colors, and I changed the default colors to a [colorblind-friendly palette](https://colorbrewer2.org/#type=qualitative&scheme=Dark2&n=3). 

Looks pretty slick:
![image](https://user-images.githubusercontent.com/2790777/151818579-14980fad-9946-4f70-a13a-b8dc814f3f30.png)

kmdalton commented:
@speleo3 -- I cleaned this pr up a bit more, added tab-complete, and fixed a bug where the B-axis cone wasn't rendering for orthorhombic space groups. See [here](https://github.com/schrodinger/pymol-open-source/issues/220) for details, but I think there is a bug in `cgo`.

speleo3 commented on psico/xtal.py:
```diff
@@ -316,12 +316,129 @@ def biomolecule(name=None, filename=None, prefix=None, number=1, suffix=None,
 cmd.auto_arg[3]['supercell'] = cmd.auto_arg[0]['pseudoatom']
+cmd.auto_arg[0]["cell_axes"] = [cmd.object_sc, "object", ""]
 
-# vi: ts=4:sw=4:smarttab:expandtab
```
can you please keep this line?

speleo3 commented on psico/xtal.py:
```diff
@@ -316,12 +316,129 @@ def biomolecule(name=None, filename=None, prefix=None, number=1, suffix=None,
     cmd.disable(name)
     cmd.group('%s_%s' % (prefix, suffix), '%s_%s_*' % (prefix, suffix))
 
+from chempy import cpv
```
please move this import to the top of the file (after `from pymol import ...`)

speleo3 commented on psico/xtal.py:
```diff
@@ -316,12 +316,129 @@ def biomolecule(name=None, filename=None, prefix=None, number=1, suffix=None,
+
+    Draw arrows corresponding to the crystallographic axes.
+
+    The default color palette is colorblind friendly but close to the familiar red, green, and blue for the a, b, and, c axes respectively.
```
I suggest to wrap this line (after "familiar").

speleo3 reviewed (changes requested):
Very nice, I like it! `cell_axes` is a good name.

One question: Do you mind if I merge this with the "squash" option? That would combine your 7 commits into one, which produces a nicer git history and works better with git blame (for my taste). For the future, I also recommend to make a named branch in your fork, don't make pull requests from your "master" branch.

kmdalton commented:
I don't mind if you squash the merge. I can make a branch next time. I believe I resolved the comments in your code review. Let me know if you need anything else from me. 

Cheers
